vividus-tests/src/main/resources/properties/suite/integration/suite.properties

```diff
@@ -12,6 +12,8 @@ bdd.batch-1.meta-filters=groovy: !skip && epic != 'vividus-plugin-ssh' && !local
 bdd.batch-1.threads=5
 bdd.batch-1.story-execution-timeout=PT10M
 bdd.variables.batch-1.locale=ca
+bdd.variables.batch-1.filtered-examples-table={transformer=FILTERING, byColumnNames=column3}\
```

Comment:
does it make sense to put some variables into this table definition? global/scoped?